src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -26,6 +26,9 @@
  */
```

Comment:
Since you're overloading this class, is it worth adding a descriptive comment? e.g. these fall into <x> categories, which are...

- types
- ?
- ?